### PR TITLE
:sparkles: feature/add-zaakobject-to-ES

### PR DESCRIPTION
**Pull request by damm89**
Original issue - https://github.com/GemeenteUtrecht/ZGW/issues/852

### CODE REVIEW

**Comment by svenvandescheur** [comment]
Also part of: https://github.com/GemeenteUtrecht/zaakafhandelcomponent/pull/406

**Review by annashamray — APPROVED** [review]
Nice PR! 
I have small remarks which are optional

File: backend/src/zac/elasticsearch/api.py [hunk 1]
```diff
@@ -42,6 +49,14 @@ def create_zaak_document(zaak: Zaak) -> ZaakDocument:
...
         status_document = None
 
+    zaakobjecten = [
+        ZaakObjectDocument(
+            url=zo.url,
+            object=zo.object,
+        )
+        for zo in get_zaakobjecten(zaak)
+    ]
+
```
[the remark this answers is not in the document]
Comment (damm89) [reply backend/src/zac/elasticsearch/api.py:59]: Yes!

File: backend/src/zac/elasticsearch/drf_api/serializers.py [hunk 2]
```diff
@@ -130,6 +134,11 @@ class StatusDocumentSerializer(serializers.Serializer):
     statustoelichting = serializers.CharField(required=False)
 
 
+class ZaakObjectDocumentSerializer(serializers.Serializer):
+    url = serializers.URLField(required=False)
+    object = serializers.URLField(required=False)
```
[the remark this answers is not in the document]
Comment (damm89) [reply backend/src/zac/elasticsearch/drf_api/serializers.py:139]: I'll start right away.

**Comment by svenvandescheur** [comment]
Merged in: https://github.com/GemeenteUtrecht/zaakafhandelcomponent/pull/406

**Pull request closed without merging**